app/presenters/idv/ssn_presenter.rb

```diff
@@ -0,0 +1,25 @@
+module Idv
+  class SsnPresenter
+    include ActionView::Helpers::TranslationHelper
+
+    attr_reader :sp_name, :ssn_form, :step_indicator_steps
+
+    def initialize(sp_name:, ssn_form:, step_indicator_steps:)
+      @sp_name = sp_name
+      @ssn_form = ssn_form
+      @step_indicator_steps = step_indicator_steps
+    end
```

Comment:
I think we need the ssn_form [here](https://github.com/18F/identity-idp/blob/54663b97fd7002dd13b7a4986058fc2c407f9240/app/views/idv/shared/ssn.html.erb#L65).

Reply:
right but at that point couldn't we access it as `@ssn_form`?

Reply:
My intention was to only need one instance variable to call show, `@ssn_presenter`. Is there a reason you prefer to have the ivars separately?

Reply:
I guess when I look at this Presenter class, I feel like it requires a lot of input variables and doesn't provide much other than re-exposing them, so I was proposing ways to simplify? In the end, one ivar or two, it's six of one, half a dozen of the other, I don't feel very strongly.

Reply:
Since the template is shared between two controllers, I was trying to corral the interface and make it obvious how to use the template. I agree that it's not obvious what the best way to do that is. Is there a way to clarify my intent?

Reply:
sometimes we leave comment blocks at the top of templates listing the required ivars

Reply:
Good point! I took out the old ivars but didn't add the new one. However this turns out, I'll make sure there's an accurate comment at the top.